diff --git a/node_modules/fastify/types/instance.d.ts b/node_modules/fastify/types/instance.d.ts
index 67d6930..60b7916 100644
--- a/node_modules/fastify/types/instance.d.ts
+++ b/node_modules/fastify/types/instance.d.ts
@@ -62,20 +62,20 @@ export interface FastifyInstance<
   getDefaultRoute: DefaultRoute<RawRequest, RawReply>;
   setDefaultRoute(defaultRoute: DefaultRoute<RawRequest, RawReply>): void;
 
-  route<
-    RouteGeneric extends RouteGenericInterface = RouteGenericInterface,
-    ContextConfig = ContextConfigDefault,
-    SchemaCompiler = FastifySchema,
-  >(opts: RouteOptions<RawServer, RawRequest, RawReply, RouteGeneric, ContextConfig, SchemaCompiler>): FastifyInstance<RawServer, RawRequest, RawReply, Logger>;
-
-  get: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
-  head: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
-  post: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
-  put: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
-  delete: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
-  options: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
-  patch: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
-  all: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
+  // route<
+  //   RouteGeneric extends RouteGenericInterface = RouteGenericInterface,
+  //   ContextConfig = ContextConfigDefault,
+  //   SchemaCompiler = FastifySchema,
+  // >(opts: RouteOptions<RawServer, RawRequest, RawReply, RouteGeneric, ContextConfig, SchemaCompiler>): FastifyInstance<RawServer, RawRequest, RawReply, Logger>;
+
+  // get: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
+  // head: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
+  // post: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
+  // put: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
+  // delete: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
+  // options: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
+  // patch: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
+  // all: RouteShorthandMethod<RawServer, RawRequest, RawReply>;
 
   // addHook: overloads
 
